File: ECnotes/Sem2/LivExamples/CS/Ch22WebParts/OrderCalculator.ascx.cs
using System;
using System.Data;
using System.Configuration;
using System.Collections;
using System.Web;
using System.Web.Security;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Web.UI.HtmlControls;

public partial class OrderCalculator : System.Web.UI.UserControl
{
    protected void btnCalculate_Click(object sender, EventArgs e)
    {
        decimal price = Convert.ToDecimal(txtPrice.Text);

        int quanity = Convert.ToInt32(txtQuantity.Text);

        decimal total = price * quanity;

        lblTotal.Text = total.ToString("c");
    }
}

File: ECnotes/Sem2/LivExamples/VB/Ch22WebParts/OrderCalculator.ascx.vb

Partial Class OrderCalculator
    Inherits System.Web.UI.UserControl

    Protected Sub btnCalculate_Click(ByVal sender As Object, _
            ByVal e As System.EventArgs) Handles btnCalculate.Click
        Dim dPrice As Decimal = CType(txtPrice.Text, Decimal)
        Dim iQuanity As Integer = CType(txtQuantity.Text, Integer)
        Dim dTotal As Decimal = dPrice * iQuanity
        lblTotal.Text = FormatCurrency(dTotal)
    End Sub
End Class
